web/src/engine/websites/templates/FlatManga.ts

```diff
@@ -0,0 +1,110 @@
+﻿import { AddAntiScrapingDetection, FetchRedirection } from "../../platform/AntiScrapingDetection";
+import { FetchCSS } from "../../platform/FetchProvider";
+import { DecoratableMangaScraper, type Chapter, type MangaScraper } from "../../providers/MangaPlugin";
+import { Page } from "../../providers/MangaPlugin";
+import * as Common from '../decorators/Common';
+
+AddAntiScrapingDetection(async (invoke) => {
+    const result = await invoke<boolean>(`document.documentElement.innerHTML.includes('ct_anti_ddos_key')`);// Sample => Mangagun, NicoManga, Rawinu, Weloma, WeloveManga
+    if (result) {
+        await new Promise(resolve => setTimeout(resolve, 3000));
+        return FetchRedirection.Automatic;
+    } else return undefined;
+
+});
+
+export function MangaLabelExtractor(element: HTMLElement) {
+    return CleanTitle(element.getAttribute('text') ? element.getAttribute('text') : element.textContent);
+}
+
+export function MangaExtractor(anchor: HTMLAnchorElement) {
+    return {
+        id: anchor.pathname,
+        title: CleanTitle(anchor.getAttribute('text') ? anchor.text : anchor.textContent)
+    };
+}
+
+export function CleanTitle(title: string): string {
+    title = title.replace(/\s*[~\-―〜]\s*RAW\s*\(MANGA\)\s*$/i, '').trim();
+    title = title.replace(/\s*[~\-―〜]\s*(MANGA)?\s*RAW\s*$/i, '').trim();
+    title = title.replace(/\(raw\)/i, '').trim();
+    return title.replace(/\(manga\)/i, '').trim();
+}
+
+export function PageLinkExtractor<E extends HTMLImageElement>(this: MangaScraper, element: E): string {
+    let page = element.dataset.aload || element.dataset.src || element.dataset.srcset || element.dataset.original || element.dataset.pagespeedLazySrc || element.src;
+    try {
+        page = window.atob(page);
+    } catch { }
+    return page.replace(/\n/g, '');
+
+}
+
+export const pathSinglePageManga = '/manga-list.html?listType=allABC';
+const pathMultiPageManga = '/manga-list.html?page={page}';
+
+export const queryMangaTitle = [
+    'li:last-of-type span[itemprop="name"]',
+    'ul.manga-info h3',
+    'ul.manga-info h1',
+].join(',');
+
+export const queryMangas = [
+    'span[data-toggle="mangapop"] a',
+    'div.media h3.media-heading a',
+    'div.container a[data-toggle="mangapop"]',
+    'div.card-body div.series-title a'
+].join(',');
+
+const queryChapters = [
+    'div#tab-chapper table tr td a.chapter',
+    'ul.list-chapters > a',
+    'div#tab-chapper div#list-chapters span.title a.chapter'
+].join(',');
+
+export const queryPages = [
+    'img.chapter-img',
+    'div#chapter-content img',
+    'div.chapter-content img'
+].join(',');
+
+export const chapterScript = `
```

Comment:
Unused export, should be added when needed

Reply:
I understand the concern but its too much a hassle to remove for i have to reflect changes on another branch where all the websites are and where its needed.

I made sure no unnecessary stuff is exported already. You cant see that because you wanted smaller PR 🤷